pytomo3d/window/filter_windows.py

```diff
@@ -213,21 +234,27 @@ def filter_windows_on_measurements(windows, measurements, measure_config):
     if len(windows) == 0:
         return {}, {}
 
-    means, stds = get_measurements_std(measurements)
+    dt_means, dt_stds, dlna_means, dlna_stds = \
+        get_measurements_std(measurements)
 
     # get the measurement bounds
     final_bounds = {}
-    for comp in means:
+    for comp in dt_means:
         print("-" * 20 + "\nComponent: %s" % comp)
         user_bound = get_user_bound(comp_config[comp])
-        std_bound = get_std_bound(means[comp], stds[comp],
-                                  comp_config[comp]["std_ratio"])
-
-        bound = [max(std_bound[0], user_bound[0]),
-                 min(std_bound[1], user_bound[1])]
+        dt_std_bound = get_std_bound(dt_means[comp], dt_stds[comp],
+                                     comp_config[comp]["std_ratio"])
+        dlna_std_bound = get_std_bound(dlna_means[comp], dlna_stds[comp],
+                                       comp_config[comp]["std_ratio"])
```

Comment:
Hi Youyi,

It seems you are using the same `std_ratio` for both `dt` and `dlna`. It is necessary to separate those two values? Or you think one value would be enough?